### DISCUSSION TITLE
UMD build

### DISCUSSION THREAD

**Discussion by vborioni-onit** [dm4t2/vue-currency-input#327] [Ideas]
Hi,
we would like to upgrade from version 1.22, but the dist does not contain the UMD build anymore, would you be so kind to add it back ?

Thank you
Valerio

**Comment by dm4t2** [comment]
Why not use CJS/ESM?

**Reply by vborioni-onit** [reply]
Hi, we do not use modules yet, for now it's just too complicated for the benefit, dropping libraries on the page is very fast and easy (and some compatibility issue that we hope will be fixed soon :D)
